recipes/protobuf-c/all/conanfile.py

```diff
@@ -49,17 +49,16 @@ def layout(self):
 
     def build_requirements(self):
         # Since the package using protobuf-c will also need to use protoc (part of protobuf),
-        # we want to make sure the protobuf dep is visible, but the visible param is only available in v2
-        # TODO: Remove after dropping Conan 1.x
-        if conan_version >= Version("2"):
-            self.tool_requires("protobuf/3.21.9", visible=True)
-        else:
-            self.tool_requires("protobuf/3.21.9")
+        # we want to make sure the protobuf dep is visible
+        self.tool_requires("protobuf/<host_version>", visible=True)
```

Comment:
Sorry, I did not remember that visible=True can not be used for visible tool_requires.
Let's keep it as it was 🙏 

```suggestion
        if Version(self.version) >= "1.5.2":
            self.requires("protobuf/[>=3.21.9 <=6.30.1]", visible=True)
        else:
            self.requires("protobuf/3.21.9", visible=True)
```